.github/analyst_guide_data_validation.md

```diff
@@ -0,0 +1,171 @@
+# Data validation
+
+This section covers the topic of **how to validate Workplace Analytics data**. Before you apply any functions to your data set and can start looking for new insights, it is recommended that you perform data validation. This best practice is applicable regardless of whether your aim is to explore the data, to establish a baseline, or to perform advanced analytics.
+
+## Why validate?
+
+There are several reasons why you should validate your Workplace Analytics data.
```

Comment:
Data validation is a crucial step for every analyst as it ensures you can trust the data you are using to be accurate, clean and helpful. Data validation ensures the information you hold in your system provides a good basis for your analyses. Making sure the data you use is correct is a proactive way to safeguard from the starting point.
//Before "there are several.."??